sql/moz-fx-data-shared-prod/telemetry_dev_cycle_derived/glean_metrics_external_v1/query.py

```diff
@@ -0,0 +1,122 @@
+"""Glean metric data - download from API, clean and upload to BigQuery."""
+
+import logging
+
+import click
+import requests
+from google.api_core.exceptions import BadRequest
+from google.cloud import bigquery
+
+API_BASE_URL = "https://probeinfo.telemetry.mozilla.org"
+SCHEMA = [
+    bigquery.SchemaField("glean_app", "STRING"),
+    bigquery.SchemaField("metric", "STRING"),
+    bigquery.SchemaField("type", "STRING"),
+    bigquery.SchemaField("first_seen_date", "DATE"),
+    bigquery.SchemaField("last_seen_date", "DATE"),
+    bigquery.SchemaField("expires", "STRING"),
+]
+PROJECT_ID = "moz-fx-data-shared-prod"
```

Comment:
I'd consider prefixing these with `DEFAULT_` since that appears to be their main purpose.